peps/pep-9999.rst

```diff
@@ -104,6 +189,7 @@ Footnotes
 .. [3] https://gmpy2.readthedocs.io/en/stable/mpz.html#gmpy2.c_div
 .. [4] https://github.com/python/cpython/issues/120950
 .. [5] https://github.com/python/cpython/issues/81313
+.. [6] :pep:`387`
```

Comment:
```suggestion
```
I think it's redundant.  PEP aren't docs and we already have refrence to the Glossary.

Reply:
Okay, I'll remove the link!